docker-compose.override.yml.dist

```diff
@@ -8,6 +8,8 @@ services:
 #  traefik:
 #    ports:
 #      - '80:80' # Uncomment to use the 80 port on your host instead of the 7500 one.
+#    volumes:
+#      - /run/user/1000/docker.sock:/var/run/docker.sock # Uncomment for docker rootless
```

Comment:
is it always a 1000 uid ?

Reply:
99% of time yes because it's the default user id. But in some very special cases, it may vary. The user will be able to edit it in its own docker-compose.override.yml file

Reply:
can `/run/user/$UID/docker.sock:/var/run/docker.sock` works here ?

Reply:
Nop cause $UID is not configured by default on many systems (like mine). I could rely on $DOCKER_HOST, but its content is invalid for docker-compose syntax:
```shell
DOCKER_HOST=unix:///run/user/1000/docker.sock
```

Reply:
Not possible, cause environment variable substitution is only available with environment variables, and UID is not an environment variable but a shell variable: https://github.com/docker/compose/issues/4189#issuecomment-320362242

Reply:
But according to [this comment](https://github.com/docker/compose/issues/2380#issuecomment-603121993):
>Default linux user have 1000 user id and 1000 group id.

Reply:
lgtm, then, you can merge it :)